### PR TITLE
Adding section "Building FreeIPA from source" on README

### PR DESCRIPTION
**Pull request by felipevolpone**
Fixes: https://pagure.io/freeipa/issue/6725

Preview of it: https://github.com/felipevolpone/freeipa/tree/readme-6725#for-developers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,12 @@ server project page at http://www.freeipa.org/ .
 The most up-to-date documentation can be found at
 http://freeipa.org/page/Documentation .
 
+## For developers
+
+* Building FreeIPA from source
+    * Please, check http://www.freeipa.org/page/Build
+    * Please, check the BUILD.txt file at the root directory
```
Comment (rcritten) [inline README.md]: I'd drop the "Please, check" from each line, to look like:

* http://www.freeipa.org/page/Build
* See the BUILD.txt file in the source root directory

**Comment by tomaskrizek** [comment]
It's a detail, but I'd prefer to see the `For developers` section below `Quick Start`.

**Comment by felipevolpone** [comment]
Done

**Comment by pvomacka** [comment]
master:

* be2fba08ce3d89448d4a87f9d0dd15f605cf6dd1 Adding section "Building FreeIPA from source" on README

**Pull request closed without merging**